README.md

```diff
@@ -263,13 +298,13 @@ You can configure the Functions Framework using command-line flags or environmen
 ## Enable Google Cloud Functions Events
 
 The Functions Framework can unmarshall incoming
-Google Cloud Functions [event](https://cloud.google.com/functions/docs/concepts/events-triggers#events) payloads to `data` and `context` objects.
+Google Cloud Functions [event](https://cloud.google.com/functions/docs/concepts/events-triggers#events) payloads to `event` and `context` objects.
 These will be passed as arguments to your function when it receives a request.
 Note that your function must use the `event`-style function signature:
 
 ```python
-def hello(data, context):
-    print(data)
+def hello(event, context):
+    print(event)
     print(context)
 ```
```

Comment:
Usually, we refer to the whole event as an `event`, (like event and cloudevent), not the first param.

The event we refer to usually has two properties, `data`, and `context`. I suggest we keep the same terms here. This is like the docs are currently, and the Node Functions Framework. Although Java uses event.

If we want to use the term event, context, I suggest we do so in a separate PR and have a plan to update the docs at the same time.

Reply:
FWIW in most other language samples we use `(message, context)`. I don't think it really matters much TBH